app/views/content/funding-and-support/salaried-teacher-training.md

```diff
@@ -108,9 +108,11 @@ Some salaried courses award QTS only. You may have the option to do a salaried c
 
 ## How do I find a salaried teacher training course? 
 
-You can [find postgraduate teacher training](https://find-teacher-training-courses.service.gov.uk/) and filter by salaried courses. 
+To apply for salaried teacher training, you can [find postgraduate teacher training](https://find-teacher-training-courses.service.gov.uk/) and filter by salaried courses. 
```

Comment:
Should we tweak the wording here re the filter, to align more with what the filter says?

![image](https://github.com/user-attachments/assets/d216f153-8914-4737-8607-45dd409c3f89)